static/js/settings_display.js

```diff
@@ -54,16 +52,23 @@ exports.set_up = function () {
 
         var context = {};
         context.lang = new_language;
+        var spinner = $("#language-settings-status").expectOne();
+        loading.make_indicator(spinner, {text: 'Saving ...'});
+        $('.loading_indicator_spinner').css("margin-top","1px");
+        $('.loading_indicator_text').css("margin-top","1px");
 
         channel.patch({
             url: '/json/settings/display',
             data: data,
             success: function () {
-                ui_report.success(i18n.t("__lang__ is now the default language!  You will need to reload the window for your changes to take effect", context),
-                                  $('#display-settings-status').expectOne());
+                ui_report.success(i18n.t("Saved. __lang__ is now the default language! Please <a>reload</a> for the change to take effect.", context),
```

Comment:
quick note, this can just be "Saved. Please .." like the rest. I.e. we can remove the "__lang__ is now the default languag" part.

Reply:
@rishig  Updated. But the test for this fails for some reason.